proxy-libs/interface-validator/interface_validator_test.go

```diff
@@ -25,10 +25,21 @@ func cloneRepositoryAndExtractExports(repoUrl string, tag string, moduleName str
 func extractProxyLibExports(libName string) []exportsExtractor.ExtractedObject {
 	srcDir, _ := filepath.Abs("../" + libName)
 	heliosExports := exportsExtractor.ExtractExports(srcDir, libName)
-	sort.Slice(heliosExports, func(i int, j int) bool { return heliosExports[i].Name < heliosExports[j].Name })
+	heliosExports = deleteByName(heliosExports, "InstrumentedSymbols")
+	sortExports(heliosExports)
 	return heliosExports
 }
 
+func deleteByName(exports []exportsExtractor.ExtractedObject, name string) []exportsExtractor.ExtractedObject {
```

Comment:
what's that?

Reply:
I've extracted a function that was already there (see below)